multi-node/aws/cmd/kube-aws/command_up.go

```diff
@@ -226,19 +329,19 @@ func initTLS(cfg *cluster.Config, dir string) (*cluster.TLSConfig, error) {
 
 	tlsConfig := cluster.TLSConfig{
 		CACertFile:        caCertPath,
-		CACert:            mustReadFile(caCertPath),
+		CACert:            mustReadFile(caCertPath).Bytes(),
```

Comment:
If you're just calling bytes() everywhere anyway, for code clarity might make sense to just return []byte